cibuildwheel/__main__.py

```diff
@@ -109,6 +116,42 @@ def main() -> None:
 
     args = parser.parse_args(namespace=CommandLineArguments())
 
+    args.package_dir = args.package_dir.resolve()
+
+    # This are always relative to the base directory, even in SDist builds
+    args.output_dir = Path(
+        args.output_dir
+        if args.output_dir is not None
+        else os.environ.get("CIBW_OUTPUT_DIR", "wheelhouse")
```

Comment:
Might be easier to add this envvar fallback to the `parser.add_argument` call i.e. `default=os.environ.get("CIBW_OUTPUT_DIR", "wheelhouse")`